packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.js

```diff
@@ -276,11 +277,11 @@ async function configure(context, defaultValuesFilename, serviceMetadata, resour
 
   print.info('');
   print.info(
-    'You can optionally add global secondary indexes for this table. These are useful when you run queries defined in a different column than the primary key.'
+    'You can optionally add global secondary indexes for this table. These are useful when you run queries defined in a different column than the primary key.',
   );
   print.info('To learn more about indexes, see:');
   print.info(
-    'http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.SecondaryIndexes'
+    'http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.SecondaryIndexes',
```

Comment:
https, all the links. Perhaps it is a regression that was fixed before.

Reply:
fixing the ones in this file. There's lots of other occurrences that can be fixed later.